tableauserverclient/server/endpoint/endpoint.py

```diff
@@ -64,9 +69,17 @@ def _make_request(
 
         logger.debug("request {}, url: {}".format(method.__name__, url))
         if content:
-            logger.debug("request content: {}".format(content[:1000]))
+            logger.debug("request content: %r", content[:1000])
```

Comment:
You should be able to to do `{!r}` for a repr format. Down with the old string substitution!

Reply:
@t8y8  I normally agree with you, f-strings wherever possible! However, in this case, it is done this way to ensure that line never actually runs unless the log level is debug, whereas an f-string or format would eagerly evaluate to only then be discarded. 

https://docs.python.org/3/howto/logging-cookbook.html#formatting-styles

Reply:
Gross, but ok :)